backends/edk2-compat/edk2-svc-validate.c

```diff
@@ -429,6 +431,8 @@ static int validateSingularESL(size_t *bytesRead, const unsigned char *esl, size
 			prlog(PR_INFO, "\tHash: ");
 			printHex(cert, cert_size);
 		}
+	}else if (varName && !strcmp(varName, "sbat")) {
+			rc = SUCCESS;
```

Comment:
We need to discuss what is contained in an sbat and if we should be checking its contents